.github/actions/phar/action.yaml

```diff
@@ -1,28 +1,17 @@
-name: 'Build the PHAR'
-description: 'Build the PHAR for the current commit'
+name: 'Build phars for all systems'
+description: 'Build phars for the current commit'
 
 runs:
   using: "composite"
   steps:
     - name: Install tools
       uses: ./.github/actions/tools
 
-    - name: Compile PHAR Linux
-      run: vendor/bin/box compile -c box.linux-amd64.json
+    - name: Compile phars
+      run: bin/castor castor:phar:build
```

Comment:
The output is really wrong with that. Can you split in 3 différents steps (like done before) ?

Reply:
Done